x-pack/plugins/maps/public/connected_components/mb_map/keydown_scroll_zoom/keydown_scroll_zoom.tsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import { EuiText, EuiTextColor } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import type { Map as MbMap, MapMouseEvent } from '@kbn/mapbox-gl';
+
+const isMac = navigator.platform.toLowerCase().indexOf('mac') >= 0;
+
+interface Props {
+  mbMap: MbMap;
+}
+
+interface State {
+  show: boolean;
+}
+
+export class KeydownScrollZoom extends Component<Props, State> {
+  private _isMounted = false;
+  private _hideTimeout: ReturnType<typeof setTimeout> | undefined;
+
+  state: State = {
+    show: false,
+  };
+
+  componentDidMount() {
+    this._isMounted = true;
+    this.props.mbMap.on('wheel', this._onWheel);
+  }
+
+  componentWillUnmount() {
+    this._isMounted = false;
+    this.props.mbMap.off('wheel', this._onWheel);
+  }
+
+  _onWheel = (event: MapMouseEvent) => {
+    if (this._hideTimeout) {
+      clearTimeout(this._hideTimeout);
+      this._hideTimeout = undefined;
+    }
+
+    if ((isMac && event.originalEvent.metaKey) || (!isMac && event.originalEvent.ctrlKey)) {
+      this.setState({ show: false });
+      return;
+    }
+
+    this.setState({ show: true });
+    this._hideTimeout = setTimeout(() => {
+      if (this._isMounted) {
+        this.setState({ show: false });
+      }
+    }, 1500);
+    event.preventDefault();
+  };
+
+  render() {
+    return (
+      <div
+        className={classNames('mapKeydownScrollZoom', {
+          // eslint-disable-next-line @typescript-eslint/naming-convention
+          mapKeydownScrollZoom__show: this.state.show,
+        })}
+      >
+        <EuiText textAlign="center">
+          <h2>
```

Comment:
I think `<h2>` is a little too large. Maybe use `<h3>` or add `size="s"` to the `EuiText` component.

Reply:
Agree. I was actually coming to exactly discuss the size/weight of the text. Maybe something to get confirmation from design, but the text feels a bit too _bold_ to me.

Reply:
I ended up using h3 and small text. Looks much better